tests_requre/service/test_api.py

```diff
@@ -240,3 +240,20 @@ def test_get_projects_releases(client, clean_before_and_after, release_model):
     assert len(response_dict) == 1
     assert response_dict[0]["tag_name"] == SampleValues.tag_name
     assert response_dict[0]["commit_hash"] == SampleValues.commit_sha
+
+
+def test_get_projects_branches(
+    client, clean_before_and_after, a_koji_build_for_branch_push
+):
+    """Test Get Project's Releases"""
+    response = client.get(
```

Comment:
Is a recording needed for this?

Reply:
I want to blame autocomplete.
This should be _a_copr_build_for_branch_push_ but I guess tests still pass because SampleValues.branch is same for both  _a_copr_build_for_branch_push_ and  _a_koji_build_for_branch_push_